HoloLensCommander/README.md

```diff
@@ -12,76 +12,94 @@ Windows Mixed Reality Commander requires:
+* Ability to clear status messages
+* Clear camera roll on device
+* Remove all side-loaded applications
+
+#### Changes
+Version 2.1706 introduced the following behavior changes:
+* Unregister devices now honors device selection (previously unregistered all devices)
+* Reconnecting to a previous session defers common application updating until all devices have been registered
+* Device list is sorted (at device registration time) by device name and address
+* Updated collection of applications to remain running when all applications are closed	
+
+#### Fixes
+The following issues have been fixed:
+* The device list now properly scrolls vertically
+* Installing an application is more likely to identify any previous version
+* Reconnecting to a previous session is now faster
+* Improved application stability in error conditions
 
 ### Key User Interface Elements
 
 #### Device Connection
 
 ![Device Connection Controls](ReadmeImages/ConnectionControls.png)
 
-##### Connect
+##### Register
 
-The Connect button reads the user name and password, then displays a dialog allowing you to target a specific device.
+The **Register** button displays a dialog allowing you to target a specific device.
 
 Before attempting to connect to a device, please be sure that the Windows Device Portal is enabled and paired with your desired credentials.
 
 ```
 Note: Windows PCs must be rebooted after enabling the Windows Device Portal.
 ```
+##### Set Credentials
+Before a device can be registered, the default Windows Device Portal credentials must be set. To do so, click the ![Set Credentials](ReadmeImages/SetCredentialsButton.png) button.
 
-![Connection Dialog](ReadmeImages/ConnectDialog.png)
+##### Restore Previous Session
+To restore the previous session (auto saved as devices are registered and unregistered), use the ![Restore Previous Session](ReadmeImages/RestorePreviousSessionButton.PNG) button.
 
-###### Address
+```
+Note: Reconnect to previous session will become disabled after a reconnection has been attempted, a session file has been loaded or the Register button has been used one or more times.
+```
 
-Enter the IP Address in the Address field. If you are connecting to a Windows PC, you must also provide the port number (typically 50443). To determine the correct address to specify:
+![Rgister Device Dialog](ReadmeImages/ConnectDialog.png)
 
-* HoloLens: Run the Settings application, select Network & Internet > Wi-Fi.
+###### Address
+Enter the IP Address in the **Address** field. If you are connecting to a Windows PC, you must also provide the port number (typically 50443). Checking the **Desktop PC** button will append the default Windows Device Portal port, if no port has been specified. 
 
+To determine the correct address to specify:
+* HoloLens: Run the Settings application, select Network & Internet > Wi-Fi.
 * Windows PC : Run the Settings application, select Update & Security > For Developers.
```

Comment:
Nit: consistency between the first word and a space or no space before the colon